packages/account/src/Components/api-token/api-token-table-row-token-cell.jsx

```diff
@@ -37,7 +37,7 @@ const ApiTokenTableRowTokenCell = ({ token, scopes }) => {
             <Popover
                 alignment='bottom'
                 classNameBubble='dc-clipboard__popover'
-                message={should_show_token ? 'Hide this token' : 'Show this token'}
+                message={localize(should_show_token ? 'Hide this token' : 'Show this token')}
```

Comment:
This is not gonna extract the string from localize.

`message={should_show_token ? localize('Hide this token') : localize('Show this token')}`

Reply:
Thanks for this remark. I'll keep it in mind :+1: